src/Layer/PointCloudLayer.js

```diff
@@ -86,6 +93,12 @@ function markForDeletion(elt) {
  * `PointsMaterial`.
  * @property {number} [mode=MODE.COLOR] - The displaying mode of the points.
  * Values are specified in `PointsMaterial`.
+ * @property {number} [minIntensityRange=0] - The minimal intensity of the
+ * layer. Changing this value will affect the material, if it has the
+ * corresponding uniform.
+ * @property {number} [maxIntensityRange=65536] - The maximal intensity of the
```

Comment:
It isn't intuitive the maximal intensity value.
`65536` is the `256x256` result , so the `intensity` is coded with a **`uint16`**. 
What it's the `intensity` specification in `Potree`, `las/laz` and `ept`?

Reply:
There are all in 16 bits, so that's why I choose this value.

Reply:
Could you normalize the intensity to have values between 0.0 and 1.0?
It's more intuitive.

Reply:
just normalize max and min

Reply:
Done

Reply:
The material property could be normalized too, like Three js properties material.

Reply:
and use [BufferAttribute.normalized](https://threejs.org/docs/index.html#api/en/core/BufferAttribute.normalized)

Reply:
Done, I normalized it in LASParser, because this was the problem

Reply:
but `bufferAttribute` isn't normalized.

Reply:
Sorry, I forgot to push the changes :sweat_smile: